pa11ycrawler/reporter.py

```diff
@@ -209,7 +209,11 @@ def make_html(self):
             result_file = os.path.join(self.results_dir, info['filename'] + '.1.0.json')
 
             with open(result_file, 'r') as report:
-                results = json.load(report)
+                try:
+                    results = json.load(report)
+                except ValueError as error:
+                    log.error(error.message)
```

Comment:
I don't think it should be needed, since the message is already something that was formatted to print in the console.